### PR TITLE
freedroidrpg, bump revision for issues libpng16/libSDL_*_1.2

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

**Comment by Begasus** [comment]
Just a quick look today, still need to disable the use of -rdynamic, also the game won't start as it can't find the configuration file: Unable to open configuration file /boot/home/.freedroid_rpg/fdrpg.cfg

File: games-rpg/freedroidrpg/freedroidrpg-0.16.1.recipe [hunk 1]
```diff
@@ -102,19 +102,6 @@ PATCH()
...
-	aclocal
-	echo "=== automake"
-	automake --add-missing
-	echo "=== autoreconf"
-	autoreconf
-	echo "=== autoconf"
-	autoconf
-	echo "=== automake"
-	automake
-	runConfigure ./configure --disable-sdltest --with-embedded-lua
```
Comment (alaviss) [inline games-rpg/freedroidrpg/freedroidrpg-0.16.1.recipe]: are you sure running configure is not necessary?

Comment (Begasus) [reply games-rpg/freedroidrpg/freedroidrpg-0.16.1.recipe]: hmm probably removed the line to go straight to make for checking :)

Comment (Begasus) [reply games-rpg/freedroidrpg/freedroidrpg-0.16.1.recipe]: Fixed :)

**Comment by Begasus** [comment]
Game doesn't start here, related to (closed) issue https://github.com/haikuports/haikuports/issues/151

**Pull request merged**